dynawo/sources/API/JOB/JOBDynModelsEntryFactory.cpp

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```

Comment:
Update the copyright to 2020 ;)

Reply:
Not a big deal, I don't even know if we have done it recently but this is nevertheless a good practice so let's try to follow it.

Reply:
good point, it is now solved